src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitController.kt

```diff
@@ -44,6 +45,7 @@ const val VISIT_BOOK: String = "$VISIT_CONTROLLER_PATH/{applicationReference}/bo
 const val VISIT_CANCEL: String = "$VISIT_CONTROLLER_PATH/{reference}/cancel"
 const val GET_VISIT_BY_REFERENCE: String = "$VISIT_CONTROLLER_PATH/{reference}"
 const val GET_VISITS_BY: String = "$VISIT_CONTROLLER_PATH/session-template"
+const val VISIT_IGNORE_NOTIFICATIONS: String = "$VISIT_CONTROLLER_PATH/{reference}/ignore-notifications"
```

Comment:
ignore-notifications or do-not-change or some other name?

Reply:
I would put this in notification controller..

Reply:
updated